File: spring-aop/src/main/java/com/study/aop/order/api/OrderApi.java
package com.study.aop.order.api;

import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RequestMapping("order")
@RestController
public class OrderApi {

    @GetMapping
    public ResponseEntity<?> getOrder() {

    }
}

File: ktspring/order-api/src/main/kotlin/study/multimodule/order/api/OrderApi.kt
package study.multimodule.order.api

import org.springframework.web.bind.annotation.PathVariable
import org.springframework.web.bind.annotation.PostMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController
import study.multimodule.order.service.OrderService

@RequestMapping("/api/orders")
@RestController
class OrderApi(
    private val orderService: OrderService
) {

    /**
     * 주문 결제 시작 API
     */
    @PostMapping("/payment/{idKey}/start")
    fun startOrderPayment(
        @PathVariable("idKey") idKey: String
    ) {
        orderService.startOrderPayment(idKey = idKey)
    }

    /**
     * 주문 결제 완료 API
     */
    @PostMapping("/payment/{idKey}/success")
    fun successOrderPayment(
        @PathVariable("idKey") idKey: String
    ) {
        orderService.completeOrderPayment(idKey = idKey)
    }
}